tests/end-to-end/external-solution-integration/internal/scenario/compass/scenario.go

```diff
@@ -25,6 +27,7 @@ func (s *Scenario) AddFlags(set *pflag.FlagSet) {
 	pflag.StringVar(&s.testID, "testID", "compass-e2e-test", "domain")
 	pflag.BoolVar(&s.skipSSLVerify, "skipSSLVerify", false, "Skip verification of service SSL certificates")
 	pflag.DurationVar(&s.eventSendDelay, "eventSendDelay", time.Duration(10)*time.Second, "Wait time in seconds before sending the first event, e.g. 5s")
+	pflag.StringVar(&s.testServiceImage, "testServiceImage", "eu.gcr.io/kyma-project/event-subscriber-tools:PR-8483", "TestServiceImage")
```

Comment:
Do we want to have `"eu.gcr.io/kyma-project/event-subscriber-tools:PR-8483"` as a default value harcoded here? I think it would be better to use some post-master image at least.

Reply:
There will be a followup PR to this one to use the final images.